spec.emu

```diff
@@ -152,12 +160,9 @@ location: https://github.com/tc39/proposal-decimal/
 	<p>If _options_ is specified and _roundingMode_ is a property of _options_, then Get(_options_, *roundingMode*) must be a rounding mode. If Get(_options_, *roundingMode*) is not a rounding mode, a *RangeError* will be thrown. The multiplication will be carried out according to the specified rounding mode.</p>
       </emu-note>
     </emu-clause>
-    <emu-clause id="sec-decimal-method-sqrt">
-      <h1>Decimal.prototype.sqrt ( [ _options_ ] )</h1>
-      <p>Returns the square root of *this* value.</p>
-      <emu-note>
-	<p>If _options_ is specified and _roundingMode_ is a property of _options_, then Get(_options_, *roundingMode*) must be a rounding mode. If Get(_options_, *roundingMode*) is not a rounding mode, a *RangeError* will be thrown. The calculation will be carried out according to the specified rounding mode.</p>
-      </emu-note>
+    <emu-clause id="sec-decimal-method-remainder">
+      <h1>Decimal.prototype.remainder ( _x_ )</h1>
+      <p>Computes the remainder of dividing *this* value by *_x_*. Returns a fresh result.</p>
```

Comment:
It would be good to specify whether this operation represents modulo (result has the sign of _x_) or remainder (result has the sign of _this_).

Reply:
Thanks, I've clarified that. (It's the latter.)